src/classes/luaCallable.cpp

```diff
@@ -17,12 +20,20 @@ LuaCallable::~LuaCallable() {
 	luaL_unref(state, LUA_REGISTRYINDEX, funcRef);
 }
 
+#ifndef LAPI_GDEXTENSION
 bool LuaCallable::compare_equal(const CallableCustom *p_a, const CallableCustom *p_b) {
+#else
+GDExtensionBool LuaCallable::compare_equal(const CallableCustom *p_a, const CallableCustom *p_b) {
+#endif
```

Comment:
A maybe less annoying way to do this would be to have a `#define` for the bool type, for example:

```
#ifdef LAPI_GDEXTENSION
#define LAPI_BOOL GDExtensionBool
#else
#define LAPI_BOOL bool
#endif
```

Then the code here could simply be:

```suggestion
LAPI_BOOL LuaCallable::compare_equal(const CallableCustom *p_a, const CallableCustom *p_b) {
```

Reply:
Yeah considering the GDExtension bool type is required I think this is the route to go. Appreciate the suggestion and all the GDExtension / godot-cpp work!

Reply:
Suggestion applied